### PR TITLE
Adds unused predator/predalien sounds.

### PR DESCRIPTION
**Pull request by carlarctg**
# About the pull request

Added two unused sounds, one roar, one death noise, for the future.

Added predalien growl to Devastate, predalien pounce noise to Pounce, and gave predalien its unique death noise.

Added a predator facehugged noise.

Added medium-strength footsteps to Predalien.

Added a pre_pounce_effects() proc.

# Explain why it's good for the game

> Added two unused sounds, one roar, one death noise, for the future.

These are some cool noises, taken from TGMC (https://github.com/tgstation/TerraGov-Marine-Corps/pull/12060 and https://github.com/tgstation/TerraGov-Marine-Corps/pull/12045) that fit in really well as Xenomorph screeches, and could be used in the future as I do not believe there's a place for them on any of the xeno castes so far.

> Added predalien growl to Devastate, predalien pounce noise to Pounce, and gave predalien its unique death noise.

These predalien noises were unused for some reason, adding them will help make the predalien scarier.

> Added a predator facehugged noise.

Ditto.

> Added medium-strength footsteps to Predalien.

This sounds SICK. It makes the Predalien feel way more terrifying to play and fight against as you can hear it stomping around on the ground. These are NOT the same foostep noises as the Queen, they're a different, unused, less-impactful version.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
soundadd: Added two unused sounds, one roar, one death noise, for the future.
soundadd: Added predalien growl to Devastate, predalien pounce noise to Pounce, and gave predalien its unique death noise.
soundadd: Added a predator facehugged noise.
soundadd: Added medium-strength footsteps to Predalien.
code: Added a pre_pounce_effects() proc.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
```
Comment (Zonespace27) [inline code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm:1]:
```suggestion
```

File: code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+/datum/action/xeno_action/activable/pounce/predalien/pre_pounce_effects(atom/A)
```
Comment (Zonespace27) [inline code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm:2]: this is passing an arg that doesn't exist on base (also is one letter)

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm (hunk 2 above)
Comment (carlarctg) [reply code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm:2]: bweh

File: code/modules/mob/living/carbon/xenomorph/abilities/general_abilities.dm [hunk 3]
```diff
@@ -241,6 +241,12 @@
 /datum/action/xeno_action/activable/pounce/proc/additional_effects_always()
 	return
 
+/**
+ * Effects to apply *inmediately* before pouncing.
```
Comment (harryob) [inline code/modules/mob/living/carbon/xenomorph/abilities/general_abilities.dm:245]: inmediately

**Review by harryob — APPROVED** [review]

File: code/modules/mob/living/carbon/xenomorph/abilities/general_abilities.dm (hunk 3 above)
Comment (carlarctg) [reply code/modules/mob/living/carbon/xenomorph/abilities/general_abilities.dm:245]: no

Comment (harryob) [reply code/modules/mob/living/carbon/xenomorph/abilities/general_abilities.dm:245]: fine

**Pull request merged**